stan/math/prim/fun/trace_gen_quad_form.hpp

```diff
@@ -68,10 +60,11 @@ inline return_type_t<TD, TA, TB> trace_gen_quad_form(
  * @throw std::domain_error if A cannot be multiplied by B or B cannot
  * be multiplied by D.
  */
-template <int RD, int CD, int RA, int CA, typename TB, int RB, int CB>
```

Comment:
`typename TB` is never used and therefore can not be deduced. Due to the bug this overload was never chosen.

Reply:
Could you also remove the doxygen `@tparam TB` line?